napari/_app_model/_submenus.py

```diff
@@ -25,6 +25,15 @@
             enablement=LLSCK.active_layer_is_image_3d,
         ),
     ),
+    (
+        MenuId.LAYERLIST_CONTEXT,
+        SubmenuItem(
+            submenu=MenuId.LAYERS_COPY_SPATIAL,
+            title=trans._('Copy scale & transforms'),
```

Comment:
```suggestion
            title=trans._('Copy scale and transforms'),
```

On my end the `&` isn't working
<img width="480" alt="image" src="https://github.com/napari/napari/assets/76622105/994313c1-e984-4ea9-bc9e-ff980f1bbda3">

If `transforms` doesn't cover everything, could do: `scale, transforms, etc.` :rofl: